utils_clip.py

```
@@ -1,13 +1,80 @@
 import pickle
 from typing import List
+import os
 
 import numpy as np
+from openai import OpenAI
 
 cache_clip = pickle.load(open("cache_clip.pkl", "rb"))
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)